Application/EdFi.Ods.AdminApi/Features/OdsInstanceContext/ReadOdsInstanceContext.cs

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using AutoMapper;
+using EdFi.Ods.AdminApi.Infrastructure;
+using EdFi.Ods.AdminApi.Infrastructure.Database.Queries;
+
+namespace EdFi.Ods.AdminApi.Features.OdsInstanceContext;
+
+public class ReadOdsInstanceContext : IFeature
+{
+    public void MapEndpoints(IEndpointRouteBuilder endpoints)
+    {
+        AdminApiEndpointBuilder.MapGet(endpoints, "/odsInstancesContexts", GetOdsInstanceContexts)
+            .WithDefaultDescription()
+            .WithRouteOptions(b => b.WithResponse<OdsInstanceContextModel[]>(200))
+            .BuildForVersions(AdminApiVersions.V2);
+
+        AdminApiEndpointBuilder.MapGet(endpoints, "/odsInstancesContexts/{id}", GetOdsInstanceContext)
+            .WithDefaultDescription()
+            .WithRouteOptions(b => b.WithResponse<OdsInstanceContextModel>(200))
+            .BuildForVersions(AdminApiVersions.V2);
+    }
+
+    internal Task<IResult> GetOdsInstanceContexts(IGetOdsInstanceContextsQuery getOdsInstanceContextsQuery, IMapper mapper, int offset, int limit)
+    {
+        var odsInstanceContextList = mapper.Map<List<OdsInstanceContextModel>>(getOdsInstanceContextsQuery.Execute(offset, limit));
+        return Task.FromResult(Results.Ok(odsInstanceContextList));
+    }
+
+    internal Task<IResult> GetOdsInstanceContext(IGetOdsInstanceContextByIdQuery getOdsInstanceContextByIdQuery, IMapper mapper, int id)
+    {
+        var odsInstanceContext = getOdsInstanceContextByIdQuery.Execute(id);
+        if (odsInstanceContext == null)
```

Comment:
@lchavarriaga 
We are already throwing error from "getOdsInstanceContextByIdQuery". Please remove the extra check. Please make sure update if any similar implementation.